devops/notebook.yml

```diff
@@ -8,6 +8,66 @@ variables:
   artifactFeed: interpret-extensions-feed
 
 steps:
+- bash: echo "##vso[task.prependpath]$CONDA/bin"
+  displayName: Add conda to PATH
+
+- bash: conda create --yes --quiet --name interp python=3.6
+  displayName: Create Anaconda environment
+
+- bash: |
+    source activate interp
+    conda install --yes --quiet --name interp pytorch-cpu torchvision-cpu -c pytorch
+  displayName: Install Anaconda packages
+
+- bash: |
+    source activate interp
+    pip install --upgrade pip
+  displayName: Upgrade pip to latest
+
+- bash: |
+    source activate interp
+    pip install -r requirements.txt
+  displayName: Install required pip packages
+
+- bash: |
+    source activate interp
+    pip install -e ./python
+  displayName: Install interpret-extensions
+
+- bash: |
+    source activate interp
+    pip install --upgrade setuptools wheel twine
+    cd ./python
+    python setup.py bdist_wheel
+    cd ..
+  displayName: Create a wheel
+
+- task: TwineAuthenticate@0
+  displayName: Configure twine authentication
+  inputs:
+    artifactFeeds: $(artifactFeed)
+
+- bash: |
+    source activate interp
+    cd ./python
+    twine upload -r $(artifactFeed) --config-file $(PYPIRC_PATH) dist/*
+    cd ..
+  displayName: Publish artifacts
+
+- bash: |
+    source activate interp
+    flake8 --max-line-length=119 --exclude=.git/,__pycache__/,dist/ .
+  displayName: 'Run flake8'
+
+- bash: |
+    source activate interp
+    python -m pytest test/ --junitxml=./TEST-TEST.xml 
```

Comment:
So we're wanting to run the regular tests before running the notebooks each time?

Reply:
updating.

Reply:
yes, notebook/ 
fixed.